packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -321,6 +323,10 @@ export default function useBlockDropZone( {
 		stopDragging,
 	} = unlock( useDispatch( blockEditorStore ) );
 
+	const { sectionRootClientId } = unlock( getSettings() );
```

Comment:
I see the performance and unit tests seems to be failing with an error `Cannot unlock an object that was not locked before.`. Could that be related to this, or is it an unrelated failure?

Reply:
There's also some discussions about it here - https://github.com/WordPress/gutenberg/pull/64141.

Reply:
Thanks @talldan. I've responded there. We need to retain the block editor setting, but it seems the implementation of keeping it private might need some work.

Reply:
I think we should look at @tjcafferkey's PR to see if we can improve the implementation of the private setting.